### ISSUE TITLE
Problem when restoring Microsoft.Azure.WebJobs.Extensions.OpenApi 0.8.0-preview

### ISSUE THREAD

**Issue by ThumNet** [Azure/azure-functions-openapi-extension#178]
Hi,

When restoring the `Microsoft.Azure.WebJobs.Extensions.OpenApi 0.8.0-preview` package we receive the following error in our Pipeline:

```
error NU1101: Unable to find package Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions. No packages exist with this id in source(s): NuGetOrg
```

It appears to be a dependency of the package:
https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions/

**Comment by justinyoo** [comment]
@ThumNet Yes, we're aware of it and in the middle of fixing it. In the meantime, please stay at 0.7.2-preview.

**Comment by justinyoo** [comment]
Related to #175

**Comment by ThumNet** [comment]
Thank you for the reply, reverted to 0.7.2.
Good luck with the investigation!

**Comment by PehrGit** [comment]
> @ThumNet Yes, we're aware of it and in the middle of fixing it. In the meantime, please stay at 0.7.2-preview.

Hi @justinyoo, any luck so far?

**Comment by joostvanhassel** [comment]
@justinyoo it seems three projects are excluded from `dotnet pack` in the `v0.8.0-preview`-branch;
https://github.com/Azure/azure-functions-openapi-extension/commit/3ac094ff8e9b5550c40c01a73b71be2841579360#diff-a54e4b4b9f4c4f4662f53f10d2d46d0764b5f6480d0124360d104139f1160576R12-R15

Would un-commenting these lines resolve the issue, or are there other issues which need to be solved?

**Comment by kharnt0x** [comment]
Noting that this is not fixed by using 0.8.1-preview

**Comment by justinyoo** [comment]
v0.8.0-preview is no longer available. Try v0.8.1-preview instead, please.
